types/shared.d.ts

```diff
@@ -129,7 +129,7 @@ export interface IStripeConnectInitParams {
   /**
    * The client secret for the connected account.
    */
-  clientSecret: string;
+  clientSecret: string | (() => Promise<string>);
```

Comment:
I think a new property name would be best here - {clientSecret: () => ...} is odd.

Reply:
I updated this to fetchClientSecret